games/minetest/Portfile

```diff
@@ -109,8 +112,104 @@ configure.args-append   -DCMAKE_BUILD_TYPE=Release \
                         -DENABLE_LUAJIT=ON \
                         -DLUA_INCLUDE_DIR:PATH=${prefix}/include/luajit-2.1 \
                         -DLUA_LIBRARY:FILEPATH=${prefix}/lib/libluajit-5.1.dylib \
+                        -USE_GPROF=FALSE \
                         -DVERSION_EXTRA=MacPorts-rev${revision}
 
+default_variants        +GLES
```

Comment:
Can't we make OpenGL ES be used only on Snow Leopard (as it is unnecessary for newer systems)? I had the same idea to do this, but never got around to testing it on a virtual machine.

Reply:
Again at https://trac.macports.org/ticket/66599

Furthermore, please be aware that upstream MT devs apparently introduced more GLES statements into new code to **upstream "irrlicht 1.9.0mt9"** only recently by: https://github.com/minetest/irrlicht/releases/tag/1.9.0mt9

Hope this helps.

Reply:
IMO there is nothing wrong with '+GLES' and one could make use of "minetest -GLES" anyway. Only proofs to me that my idea of presenting GLES by variant was a good idea.

GLES stays until further notice.